tests/storage_test.go

```diff
@@ -199,7 +199,7 @@ var _ = Describe("Storage", func() {
 					&expect.BSnd{S: "sudo mkfs.ext4 /dev/vdc\n"},
 					&expect.BExp{R: "\\$ "},
 					&expect.BSnd{S: "echo $?\n"},
```

Comment:
there are some places with Snd \n still
is it fine ?

for me it was fine when i worked on it, i guess something delicate was changed

Reply:
Batches that are not using the newly introduced method `ExpectBatchWithSendValidation` can keep using the `\n`.
However, `ExpectBatchWithSendValidation` is not working with `\n`, since `\n` is translated to `\r\n` which causes the send validation to fail.

I agree it is confusing. I will add a commit to the PR introducing an LD (line delimiter).

Reply:
can you enforce that there isnt just \n alone , or even replace it with \r ?